app/views/support_interface/settings/feature_flags.html.erb

```diff
@@ -28,10 +28,8 @@
   ].compact
   ) do %>
     <%= render SummaryCardHeaderComponent.new(title: feature_name.humanize, heading_level: 3) do %>
-      <% if FeatureFlag.active?(feature_name) %>
-        <%= button_to 'Deactivate', support_interface_deactivate_feature_flag_path(feature_name), class: 'govuk-button app-button--tertiary govuk-!-margin-bottom-0' %>
-      <% else %>
-        <%= button_to 'Activate', support_interface_activate_feature_flag_path(feature_name), class: 'govuk-button app-button--tertiary govuk-!-margin-bottom-0' %>
+      <%= button_to support_interface_deactivate_feature_flag_path(feature_name), class: 'govuk-button app-button--tertiary govuk-!-margin-bottom-0' do %>
+        <%= FeatureFlag.active?(feature_name) ? 'Deactivate' : 'Activate' %><span class="govuk-visually-hidden"> ‘<%= feature_name.humanize %>’ feature</span>
```

Comment:
Extra hidden label looks good but I think this is always going to link to the `support_interface_deactivate_feature_flag_path` and not the `support_interface_activate_feature_flag_path` that logic needs to be wrapped in a ternary conditional or reverted back to the long-hand `if-else-end`.

Reply:
Argh! Good spot!